stock_reserve/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
```

Comment:
Replace by shortened comments like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py